src/redux/categories/categories.js

```diff
@@ -1,13 +1,20 @@
-const CHECK_CATEGORY = 'CHECK_CATEGORY';
-
-const initialState = 'UNDER CONSTRUCTION';
-export default function categoriesReducer(state = initialState, action) {
-  switch (action.type) {
-    case CHECK_CATEGORY:
-      return state;
-    default:
-      return state;
-  }
-}
-
-export const checkCategory = () => ({ type: CHECK_CATEGORY });
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: '',
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    setCategorie: (state) => {
+      state.categories = 'COMMING SOON';
+    },
+  },
+});
```

Comment:
- [ ] Please create a button for checking status from the `<Categories />` component. This button would use a function checkCategoriesStatus (name can be different) made in the component to dispatch the corresponding action creator 👍

Reply:
Well resolved! ✔️